src/sass/gi/partials/_gi-modal.scss

```diff
@@ -1,12 +1,21 @@
 .gi-app {
-  
+
   .va-modal {
     background-color: rgba($color-gray-dark, .8);
     padding: 10px;
 
+    @include media-maxwidth($small-screen) {
+      overflow-y: scroll;
+    }
```

Comment:
Should these be specific to GIBCT? I would think this issue can affect modals in general.